elementary/monitor/dbt_project/macros/alerts/population/model_alerts.sql

```diff
@@ -61,63 +67,39 @@
         select * from {{ ref('snapshot_run_results') }}
     ),
 
-    all_alerts as ( 
-        select 
-            model_execution_id,
-            model_execution_id as alert_id,
-            unique_id,
-            invocation_id,
-            name,
-            generated_at,
-            status,
-            full_refresh,
-            message,
-            execution_time,
-            execute_started_at,
-            execute_completed_at,
-            compile_started_at,
-            compile_completed_at,
-            compiled_code,
-            database_name,
-            schema_name,
-            materialization,
-            tags,
-            package_name,
-            path,
-            original_path,
-            owner,
-            alias 
-        from model_run_results
-        where lower(status) != 'success'
-        and {{ elementary.edr_cast_as_timestamp('generated_at') }} > {{ elementary.edr_timeadd('day', -1 * days_back, elementary.edr_current_timestamp()) }}
-        union all
-        select 
-            model_execution_id,
+    seed_run_results as (
```

Comment:
Can you make sure to render those only in the `seed_run_results` actually exists?
You can see an example of how to do so [here](https://github.com/elementary-data/elementary/blob/7d3deefb5e690822a2e791c7310aaf5072983ffb/elementary/monitor/dbt_project/models/alerts/deprecated/alerts_models.sql#L13).

Essentially define the relation at the top of the model, and then wrap its usages with `{% if seed_run_results_relation %}`.

Reply:
We shouldn't add it, as the safety is intended to handle cases where an older dbt package version is used (that doesn't yet have `seed_run_results`)